src/app/index.ts

```diff
@@ -233,6 +240,7 @@ module.exports = class extends yo {
       usageDataObject.reportEvent(defaults.promptSelectionstEventName, projectInfo);
     } catch (err) {
       usageDataObject.reportError(defaults.promptSelectionsErrorEventName, new Error('Prompting Error: ' + err));
+      throw err;
```

Comment:
What's the side effect of throwing the error when we didn't before (we were eating the error)?

Reply:
Yo office would continue the process saying it was going to download an `undefined` project for host `undefined`.
That download would fail and wouldn't throw anything meaningful. So I thought we should thrown the error

Reply:
If that was the experience before, then what's the experience when throwing the error?

Reply:
It would simply throw the error for the client and terminate the process without trying to download and fail then.
The error thrown is any error message we can write to customers. The way it was, customer wouldn't see any errors we wrote

Reply:
Sure, just removed this line